### PR TITLE
Feature/297 remove color gradient

### PR DESCRIPTION
**Pull request by nilsml**
Remove color gradient for casing and replace with solid color

### CODE REVIEW

**Comment by nilsml** [comment]
Removed some unused vars. Anything else we talked about during code review that should be changed @ooystein?

File: package-lock.json [hunk 1]
```diff
@@ -6914,6 +6914,16 @@
       "integrity": "sha512-1Yj8h9Q+QDF5FzhMs/c9+6UntbD5MkRfRwac8DoEm9ZfUBZ7tZ55YcGVAzEe4bXsdQHEk+s9S5wsOKVdZrw0tQ==",
       "dev": true
     },
+    "bindings": {
```
Comment (ooystein) [inline package-lock.json:6917]: Should we remove this from this PR?
If we need to update we can do in another PR

**Review by ooystein — APPROVED** [review]
